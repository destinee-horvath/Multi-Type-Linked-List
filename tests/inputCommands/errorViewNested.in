VIEW-NESTED
VIEW NESTED 3
VIEW-NESTED -3
VIEW-NESTED abd
