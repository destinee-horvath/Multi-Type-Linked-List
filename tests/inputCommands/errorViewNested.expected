INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: VIEW-NESTED
